src/main_interface.jl

```diff
@@ -16,13 +16,11 @@ these fallback methods will not change even if new fallbacks are defined.
 struct HashVersion{V}
     function HashVersion{V}() where {V}
         if !(V == 3)
-            Base.depwarn("HashVersion{V} for V < 3 is deprecated, favor " *
-                         "`HashVersion{3}` in all cases where backwards compatible " *
-                         "hash values are not required.", :HashVersion)
-            @warn "Hash version 1 and 2 have been found to depend on the module in "*
-                "which they are first executed (cf [#64](https://github.com/beacon-"*
-                "biosignals/StableHashTraits.jl/pull/64)); to avoid hash instability,"*
-                " use version 3."
+            @warn "Hash version 1 and 2 are deprecated. They have been found to depend on " *
```

Comment:
I thought the depwarn made sense, why was it removed?

Reply:
Oh! Sorry, that's how I interpreted your comment 

> I'd suggest you throw this as a @warn inside stable_hash as well if 1 or 2 are used

Since the way that 1 or 2 are used is if you call `HashVersion{1/2}()`. 

I don't think it makes sense to have both a warning and a deprecation warning (since the latter gets printed in strictly fewer places than `@warn`).

I feel like the brokenness here is pretty unfortunately, and so I'm not sure how strongly to warn off users in using 1 or 2, and I took your suggestion to be that maybe we should let folks know anytime it gets used. But that could be pretty noisy if it is used by some upstream package.

Reply:
ah, fair enough. I think `depwarn` is sufficient IMO, as there's also information in the README explaining more. But I'll leave it up to you!